portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,12 +25,30 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+   
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String quote="Hello Pranjali!";
-   
-    response.setContentType("text/html;");
+    //Create ArrayList<String> object
+    ArrayList<String> list= new ArrayList<String>();   
```

Comment:
1) Change this to List<String> list = .... List is the interface and ArrayList is the implementation. It is better to use the interface to represent vars. You only need the methods exposed by the interface.
2) Use a more descriptive variable name. Maybe quotes?

Reply:
I tried using List<String> list = new ArrayList<String>();
But it wouldn't work. I get "incompatible types: java.util.List<java.lang.String>
 cannot be converted to java.util.ArrayList<java.lang.String>" error

Reply:
Alright, fixed it